packages/server/test/unit/args_spec.coffee

```diff
@@ -52,6 +52,13 @@ describe "lib/util/args", ->
       options = @setup("--run-project", "foo", "--spec", "'cypress/integration/foo_spec.js'")
       expect(options.spec[0]).to.eq("#{cwd}/cypress/integration/foo_spec.js")
 
+  context "--tag", ->
+    it "converts to array", ->
+      options = @setup("--run-project", "foo", "--tag", "nightly,production,build")
+      expect(options.tag[0]).to.eq("nightly")
```

Comment:
could we use single assertion here for complete message on error
```js
expect(options.tag).to.deep.equal(['nightly', 'production', 'build'])
```

Reply:
yes